compiler/src/iree/compiler/Codegen/Dialect/VectorExt/IR/VectorExtOps.td

```diff
@@ -40,21 +40,31 @@ def IREEVectorExt_ToLayoutOp : IREEVectorExt_PureOp<"to_layout", [
   let arguments = (ins
     AnyShaped:$input,
     VectorLayoutInterface:$layout,
-    DefaultValuedAttr<UnitAttr, "false">:$shared_memory_conversion
+    DefaultValuedAttr<UnitAttr, "false">:$shared_memory_conversion,
+    // TODO: Solve cmake IREEGPU and VectorExt cyclic dependency to
+    // change mma_Kind type to be of MMAInterfaceAttr.
+    OptionalAttr<AnyAttr>:$mma_kind
   );
   let results = (outs
     AnyShaped:$output
   );
   let builders = [
     OpBuilder<(ins "Value":$input,
                    "VectorLayoutInterface":$layout,
+                   "Attribute":$mma_kind_attr,
                    CArg<"bool", "false">:$shared_memory_conversion), [{
```

Comment:
If you swap the order of mma_kind and shared_memory_conversion in the attribute definition, you shouldn't need this constructor.